Paccy10/olympus-api
# Feature(categories): delete category

**What does this PR do?**

- Delete category

**Description of Task to be completed?**

- An authenticated admin user can delete category

**How should this be manually tested?**

- Checkout to the branch `ft-delete-category`
- Run the app with `flask run`
- Go to postman and send a `DELETE` request to /categories/{name} and replace `name` with the category name
- The named category should be deleted

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
